### DISCUSSION TITLE
[deleted]

### DISCUSSION THREAD

**Discussion by mg0x7BE** [dailydotdev/daily#1228] [General]
[deleted]

**Comment by idoshamun** [comment]
Hi @MaxGripe,

I'm sad to hear your experience, but would like to suggest a solution for you :)
If you open your feed settings and navigate to "Manage Content" you can remove listicles from your feed.
We're actively working on detecting clickbaits and removing low quality content from the feed.
You're feedback is highly valuable.

<img width="1074" alt="image" src="https://github.com/dailydotdev/daily/assets/1993245/85d76857-5d1b-4713-9164-8ff6d0419189">

**Reply by idoshamun** [reply]
Yes! You made my day now
